tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -245,62 +230,69 @@
         },
         disabledTabs: String,
       },
+      observers: [
+        '_ensureCurrentDashboardStamped(_dashboardContainersStamped, mode)',
+      ],
       _isTabEnabled: function(tab) {
         if (this.disabledTabs != null &&
             this.disabledTabs.split(',').indexOf(tab) >= 0) {
           return false;
         }
         return true;
       },
-      _getModeFromIndex: function(modeIndex) {
-        var mode = this.tabs[modeIndex];
-        setString(TAB, mode, /*useLocalStorage=*/false);
-        return mode;
-      },
-      _isReloadDisabled: function(mode) {
-        return !this._debuggerDataEnabled && this._modeIsGraphs(mode);
-      },
-      _modeIsScalars: function(mode) {
-        return mode === "scalars";
-      },
-      _modeIsImages: function(mode) {
-        return mode === "images";
-      },
-      _modeIsAudio: function(mode) {
-        return mode === "audio";
+      _getModeFromIndex(tabs, modeIndex) {
+        return tabs[modeIndex];
       },
-      _modeIsGraphs: function(mode) {
-        return mode === "graphs";
+      _getDisplayForMode(mode, dashboard) {
+        return mode === dashboard ? 'block' : 'none';
```

Comment:
When is mode set to dashboard? This doesn't match the comment describing the mode string.